src/app/screens/PeerlyScreens/DetailsScreen/index.tsx

```diff
@@ -1,33 +1,99 @@
+import SuccessIcon from '../../../../assets/peerly/svg/Vector.svg';
+
+const AppreciationDetailsScreen = ({route}) => {
+  const {cardId, appriciationList} = route.params;
+  const cardDetails = appriciationList.find(item => item.id === cardId);
+  const [reward, setReward] = useState(0);
+  const [reason, setReason] = useState('');
+  const [isObjectionModalVisible, setObjectionModalVisible] = useState(false);
+
+  const {
+    mutate: postReward,
+    isLoading: isLoadingPostReward,
+    isSuccess: isSuccessPostReward,
+    reset: resetPostReward,
+  } = usePostReward();
+
+  const {
+    mutate: postObjection,
+    isLoading: isLoadingPostObjection,
+    isSuccess: isSuccessPostObjection,
+    reset: resetPostObjection,
+  } = usePostObjection();
+
+  useEffect(() => {
+    if (cardDetails) {
+      setReward(cardDetails.given_reward_point);
+    }
+  }, [cardDetails]);
+
+  useEffect(() => {
+    if (isSuccessPostObjection && isObjectionModalVisible) {
+      setObjectionModalVisible(false);
+    }
+  }, [isSuccessPostObjection, isObjectionModalVisible]);
+
+  const handleReward = (point: number) => {
+    const payload = {
+      params: {
+        id: cardDetails.id,
+      },
+      body: {point: point},
+    };
+    postReward(payload);
+  };
+
+  const handleObjectionReason = () => {
+    const payload = {
+      params: {
+        id: cardDetails.id,
+      },
+      body: {reporting_comment: reason},
+    };
+    postObjection(payload);
+  };
 
-const AppreciationScreen = () => {
   return (
     <View style={styles.container}>
       <View style={{alignItems: 'center'}}>
         <Image
-          source={require('../../../../assets/images/profile.png')}
+          source={
+            cardDetails?.receiver_image_url
+              ? {uri: cardDetails.receiver_image_url}
+              : require('../../../../assets/images/profile.png')
+          }
           style={styles.avatar}
         />
       </View>
       <View style={{alignItems: 'center', position: 'absolute', right: 80}}>
         <Image
-          source={require('../../../../assets/images/profile.png')}
+          source={
+            cardDetails?.sender_image_url
+              ? {uri: cardDetails.sender_image_url}
+              : require('../../../../assets/images/profile.png')
+          }
           style={[styles.smallAvatar]}
         />
       </View>
       <View style={{justifyContent: 'center', alignItems: 'center'}}>
         <View style={styles.info}>
-          <Text style={styles.name}>Mangesh Pawar</Text>
-          <Text style={styles.title}>Technical Lead</Text>
+          <Text style={styles.name}>
+            {`${cardDetails.receiver_first_name} ${cardDetails.receiver_last_name} `}
+          </Text>
+          <Text style={styles.title}>{cardDetails.receiver_designation}</Text>
         </View>
 
-        <Text style={styles.tag}>Technical Excellence</Text>
+        <Text style={styles.tag}>{cardDetails.core_value_name}</Text>
```

Comment:
lets convert fields to camel case before using it in react code.

Reply:
Sure I will made these changes